packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -224,7 +191,8 @@ module.exports = ctx => ({
       { text: 'Guides', link: '/docs/guides/' },
       { text: 'Concepts', link: '/docs/concepts/' },
       { text: 'Reference', link: '/docs/reference/' },
-      { text: 'Languages & SDKs', link: '/code/' }
+      { text: 'Languages & SDKs', link: '/code/' },
+      { text: 'Release Notes', link: '/docs/release-notes/' }
```

Comment:
Updated so the mobile nav matches the desktop left nav

Reply:
thanks!